src/ExpenseTrackerGrupo2/Controllers/BudgetController.cs

```diff
@@ -0,0 +1,135 @@
+using Microsoft.AspNetCore.Mvc;
+using ExpenseTrackerGrupo2.Business.Services.Interfaces;
+using ExpenseTrackerGrupo2.Business.Services.Mappers.Requests.Budgets;
+
+namespace ExpenseTrackerGrupo2.API.Controllers;
+
+[ApiController]
+[Route("api/v1/budget/[controller]")]
```

Comment:
![image](https://github.com/user-attachments/assets/6ab2fb73-68dc-4eb5-aded-d5a33b1e5a32)

that it only contains either budget or Budget but that it does not contain both lowercase or uppercase letters, it is better if you put the uppercase letter.